build/dist/docker-compose-ftp.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  ftp:
+    build:
+      context: build/dist/ftp
+    volumes:
+      - dockerdev-workspace-volume:/data
+    network_mode: bridge
+    ports:
```

Comment:
leave the ports section out, this would expose your machine to possible threats..
This `ports` section opens the port `21` and passive ports on your external hardware if a firewall isn't setup properly.